### DISCUSSION TITLE
how build vs solution

### DISCUSSION THREAD

**Discussion by urmas69** [PowerShell/PowerShell#18014] [General]
how build solution with visual studio 2022 preview (17.4.0 preview 1.0)
all resources are not generated ??
for example and so on
Microsoft.PowerShell.CoreCLR.Eventing\DotNetCode\Eventing\EventDescriptor.cs(47,67,47,88): error CS0103: The name 'DotNetEventingStrings' does not exist in the current context

**Comment by iSazonov** [comment]
You should use command line to build - open pwsh console, import module Build.psm1, run Start-PSBuild (with parameters you need).
After that you can use solution in VS for some helper operations.
